diddoc.go

```diff
@@ -17,7 +15,7 @@ type DocService struct {
 
 type Doc struct {
 	ID                 string                  `json:"id"`
-	AlsoKnownAs        []string                `json:"alsoKnownAs,omitempty"`
-	VerificationMethod []DocVerificationMethod `json:"verificationMethod,omitempty"`
-	Service            []DocService            `json:"service,omitempty"`
+	AlsoKnownAs        []string                `json:"alsoKnownAs"`
+	VerificationMethod []DocVerificationMethod `json:"verificationMethod"`
+	Service            []DocService            `json:"service"`
```

Comment:
Flagging this for review. I changed this because I prefer to see empty arrays in the output vs having them omitted entirely - unsure if there was any deeper reason/requirement for the original choice.

Reply:
(also plc.directory represents empty arrays this way too, e.g. https://plc.staging.bsky.dev/did:plc:q7ufzdtkkxusdqmphjimrtc4 )